tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -2875,4 +2875,35 @@ public function insert_after( $new_html ) {
 			'Should have properly applied the update from in front of the cursor.'
 		);
 	}
+
+	/**
+	 * @ticket TBD
+	 *
+	 * @covers ::remove_class
+	 */
+	public function test_remove_class_removes_leading_white_space() {
+		$processor = new WP_HTML_Tag_Processor( '<div class="remove keep">' );
+		$processor->next_tag();
+		$processor->remove_class( 'remove' );
+		$this->assertSame(
+			'<div class="keep">',
+			$processor->get_updated_html(),
+			'::remove_class did not correctly clean whitespace from removed class.'
+		);
+	}
+	/**
+	 * @ticket TBD
+	 *
+	 * @covers ::remove_class
+	 */
+	public function test_remove_class_maintains_trailing_white_space() {
+		$processor = new WP_HTML_Tag_Processor( '<div class="keep          remove">' );
+		$processor->next_tag();
+		$processor->remove_class( 'remove' );
+		$this->assertSame(
+			'<div class="keep">',
+			$processor->get_updated_html(),
+			'::remove_class did not correctly clean whitespace from removed class.'
+		);
+	}
```

Comment:
same comment here as above. I don't want to test these behaviors because these are specifically supposed to not matter.

if anything, if we want to test anything here, we should test performance metrics or diff size and leave the incidental and semantically neutral details out of a specification, or we risk making valid changes fail tests.